Makefile

```
@@ -232,3 +228,10 @@ test-e2e:
 .PHONY: test-doc-automation
 test-doc-automation:
 	$(RUN_GINKGO) $(GINKGO_FLAGS_ONE) --junit-report="test-doc-automation.xml"  tests/documentation/...
+
+
+# Generate OpenAPISpec library based on ododevapispec.yaml inside pkg/apiserver-gen; this will only generate interfaces
+# Actual implementation must be done inside pkg/apiserver-impl
+.PHONY: generate-apiserver
+generate-apiserver: ## Generate OpenAPISpec library based on ododevapispec.yaml inside pkg/apiserver-gen; also go fmt the files
+	podman run --rm -v ${PWD}:/local openapitools/openapi-generator-cli generate -i /local/ododevapispec.yaml -g go-server -o /local/pkg/apiserver-gen --additional-properties=outputAsLibrary=true,onlyInterfaces=true && echo "Formatting generated files:" && go fmt ./pkg/apiserver-gen/...
```

Can you also pin the image version, to make sure we all use the same version of the generator?

```suggestion
	podman run --rm -v ${PWD}:/local docker.io/openapitools/openapi-generator-cli:v6.6.0 generate -i /local/ododevapispec.yaml -g go-server -o /local/pkg/apiserver-gen --additional-properties=outputAsLibrary=true,onlyInterfaces=true && echo "Formatting generated files:" && go fmt ./pkg/apiserver-gen/...
```